{
    "CVE_data_meta": {
        "ASSIGNER": "cve@mitre.org",
        "ID": "CVE-2020-24583",
        "STATE": "PUBLIC"
    },
    "affects": {
        "vendor": {
            "vendor_data": [
                {
                    "product": {
                        "product_data": [
                            {
                                "product_name": "n/a",
                                "version": {
                                    "version_data": [
                                        {
                                            "version_value": "n/a"
                                        }
                                    ]
                                }
                            }
                        ]
                    },
                    "vendor_name": "n/a"
                }
            ]
        }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
        "description_data": [
            {
                "lang": "eng",
                "value": "An issue was discovered in Django 2.2 before 2.2.16, 3.0 before 3.0.10, and 3.1 before 3.1.1 (when Python 3.7+ is used). FILE_UPLOAD_DIRECTORY_PERMISSIONS mode was not applied to intermediate-level directories created in the process of uploading files. It was also not applied to intermediate-level collected static directories when using the collectstatic management command."
            }
        ]
    },
    "problemtype": {
        "problemtype_data": [
            {
                "description": [
                    {
                        "lang": "eng",
                        "value": "n/a"
                    }
                ]
            }
        ]
    },
    "references": {
        "reference_data": [
            {
                "url": "https://docs.djangoproject.com/en/dev/releases/security/",
                "refsource": "MISC",
                "name": "https://docs.djangoproject.com/en/dev/releases/security/"
            },
            {
                "refsource": "MISC",
                "name": "https://groups.google.com/forum/#!topic/django-announce/zFCMdgUnutU",
                "url": "https://groups.google.com/forum/#!topic/django-announce/zFCMdgUnutU"
            },
            {
                "refsource": "MISC",
                "name": "https://groups.google.com/forum/#!topic/django-announce/Gdqn58RqIDM",
                "url": "https://groups.google.com/forum/#!topic/django-announce/Gdqn58RqIDM"
            },
            {
                "refsource": "MISC",
                "name": "https://www.openwall.com/lists/oss-security/2020/09/01/2",
                "url": "https://www.openwall.com/lists/oss-security/2020/09/01/2"
            },
            {
                "refsource": "MISC",
                "name": "https://www.djangoproject.com/weblog/2020/sep/01/security-releases/",
                "url": "https://www.djangoproject.com/weblog/2020/sep/01/security-releases/"
            },
            {
                "refsource": "UBUNTU",
                "name": "USN-4479-1",
                "url": "https://usn.ubuntu.com/4479-1/"
            }
        ]
    }
}